### ISSUE TITLE
History of sarcoidosis

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9995]
**Preferred term label:**
History of sarcoidosis

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Sporotrichosis

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
We would model past medical history of other diseases using GA4GH Phenopackets and Mondo identifiers for the disease. I would like to avoid adding terms like this to HPO to avoid a combinatorial explosion.
